cpp/ql/lib/change-notes/2024-12-03-remove-dataflow-config-class-api.md

```diff
@@ -0,0 +1,4 @@
+---
+category: breaking
+---
+* Deleted the old deprecated data flow api that was based on extending a configuration class.
```

Comment:
Something like this will help any remaining users migrate.
```suggestion
* Deleted the old deprecated data flow API that was based on extending a configuration class. See https://github.blog/changelog/2023-08-14-new-dataflow-api-for-writing-custom-codeql-queries for instructions on migration your queries to use the new API.
```

Reply:
done